Common/include/Gauss_Elimination_core.hpp

```diff
@@ -0,0 +1,141 @@
+    su2double d0,d1,d2,d3,d4;
```

Comment:
I wrote it as include file to be include below the header. This provides the advantage to use the same modifications for the LU-Decomp. and the ILU while not blowing up the original source code by that many lines.